src/encoded/static/components/cart/add_multiple.js

```diff
@@ -128,39 +128,126 @@ CartAddAllComponent.propTypes = {
     session: PropTypes.object,
 };
 
-CartAddAllComponent.defaultProps = {
+CartAddAllSearchComponent.defaultProps = {
     inProgress: false,
     savedCartObj: null,
     session: null,
 };
 
-const mapStateToProps = (state, ownProps) => ({
+CartAddAllSearchComponent.mapStateToProps = (state, ownProps) => ({
     elements: state.elements,
     savedCartObj: state.savedCartObj,
     inProgress: state.inProgress,
     searchResults: ownProps.searchResults,
     session: ownProps.session,
 });
-const mapDispatchToProps = (dispatch, ownProps) => ({
+CartAddAllSearchComponent.mapDispatchToProps = (dispatch, ownProps) => ({
     addAllResults: elementsForCart => dispatch(addMultipleToCartAndSave(elementsForCart, !!(ownProps.session && ownProps.session['auth.userid']), ownProps.fetch)),
     setInProgress: enable => dispatch(cartOperationInProgress(enable)),
 });
 
-const CartAddAllInternal = connect(mapStateToProps, mapDispatchToProps)(CartAddAllComponent);
+const CartAddAllInternal = connect(CartAddAllSearchComponent.mapStateToProps, CartAddAllSearchComponent.mapDispatchToProps)(CartAddAllSearchComponent);
```

Comment:
Can we change "CartAddAllInternal" to be "CartAddAllSearchInternal"? Just for consistency.
We currently have:
- CartAddAllSearchComponent
- CartAddAllInternal
- CartAddAllSearch
for adding all elements, and then:
- CartAddAllElementsComponent
- CartAddAllElementsInternal
- CartAddAllElements
for adding elements from an array of datasets.